### PR TITLE
Fixed TRUST_DOMAIN in setup_x509 

### PR DESCRIPTION
**Pull request by mambelli**
Fixed setup_x509 to be successful w/ TRUST_DOMAIN set in the as Factory or Frontend parameter

### CODE REVIEW

File: creation/web_base/setup_x509.sh [hunk 1]
```diff
@@ -441,7 +441,9 @@ _main() {
         gconfig_add GLIDEIN_CONDOR_TOKEN "$GLIDEIN_CONDOR_TOKEN"
         gconfig_add GLIDEIN_CONDOR_TOKEN_ORIG "$GLIDEIN_CONDOR_TOKEN_ORIG"
         out=$(gconfig_get TRUST_DOMAIN "${glidein_config}")
-        if [[ -z "$out" ]]; then
+        if [[ -n "$out" ]]; then
+            cred_updated+=idtoken,
```
Comment (namrathaurs) [inline creation/web_base/setup_x509.sh:445]: Is it intended to have the comma at the end of the line `cred_updated+=idtoken,`  ?

**Review by namrathaurs — COMMENTED** [review]
Just a comment as I was not sure of the exact behavior/reason behind adding the comma if that's essential.

File: creation/web_base/setup_x509.sh (hunk 1 above)
Comment (mambelli) [reply creation/web_base/setup_x509.sh:445]: yes, it has to, it is a list of values

**Comment by mambelli** [comment]
resolved comments, merging

**Pull request merged**